signac/syncutil.py

```diff
@@ -160,22 +111,22 @@ class _FileModifyProxy:
 
     Parameters
     ----------
-    root :
+    root : str, optional
```

Comment:
should we change this to path?

Reply:
I vote no because this is not necessarily the Project's root/path. In this instance, it is the workspace: https://github.com/glotzerlab/signac/blob/a924bad99c1f0601f9f7ade16101fc2ec0f5e871/signac/sync.py#L502-L503